modules/library/coverage/src/main/java/org/geotools/resources/image/ExtendedImageParam.java

```diff
@@ -1,36 +0,0 @@
-/*
- *    GeoTools - The Open Source Java GIS Toolkit
- *    http://geotools.org
- *
- *    (C) 2016, Open Source Geospatial Foundation (OSGeo)
- *
- *    This library is free software; you can redistribute it and/or
- *    modify it under the terms of the GNU Lesser General Public
- *    License as published by the Free Software Foundation;
- *    version 2.1 of the License.
- *
- *    This library is distributed in the hope that it will be useful,
- *    but WITHOUT ANY WARRANTY; without even the implied warranty of
- *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- *    Lesser General Public License for more details.
- */
-package org.geotools.resources.image;
-
-import javax.imageio.ImageReadParam;
-
-/**
- * Utility class that should be used to pass extra parameters to the NetCdf image reader machinery.
- */
-public class ExtendedImageParam extends ImageReadParam {
```

Comment:
Cannot remove a public class all of a sudden, should be deprecated with an indication of what to use instead